xarray/tests/test_backends.py

```diff
@@ -5577,3 +5577,8 @@ def test_raise_writing_to_nczarr(self, mode):
                 KeyError, match="missing the attribute `_ARRAY_DIMENSIONS`,"
             ):
                 ds.to_zarr(tmp, mode=mode)
+
+
+def test_pickle_open_mfdataset_dataset():
+    ds = open_example_mfdataset(["bears.nc"])
+    assert pickle.dumps(ds)
```

Comment:
```suggestion
    assert_identical(ds, pickle.loads(pickle.dumps(ds)))
```

This would actually test whether we can restore.

Reply:
Excellent, thanks for improving the test. I was also very unsure how to use `assert`, and did not consider the reloading of the pickled object.